conf/api.go

```diff
@@ -360,7 +360,10 @@ func (confMap ConfMap) Dump() (buf string) {
 		confSectionNameSliceIndex int
 	)
 
-	buf = ""
+	// 1 Mibyte should be more than enough to hold the confMap so get the
+	// memory in 1 allocation
+	var buf strings.Builder
```

Comment:
As ever... when you modify an existing code block... it would be very helpful to follow the convention of the code being modified.